ksa_compliance/ksa_compliance/doctype/zatca_business_settings/zatca_business_settings.json

```diff
@@ -288,8 +288,9 @@
   {
    "description": "The URL for the ZATCA server to use. Use <pre style=\"display:inline;\">https://gw-fatoora.zatca.gov.sa/e-invoicing/developer-portal/</pre> for the ZATCA sandbox (testing)",
    "fieldname": "fatoora_server_url",
-   "fieldtype": "Data",
-   "label": "Fatoora Server URL"
+   "fieldtype": "Select",
+   "label": "Fatoora Server URL",
+   "options": "Sandbox | https://gw-fatoora.zatca.gov.sa/e-invoicing/developer-portal/\nSimulation | https://gw-fatoora.zatca.gov.sa/e-invoicing/simulation/\nProduction | https://gw-fatoora.zatca.gov.sa/e-invoicing/core/"
```

Comment:
I don't like this approach because it makes localization difficult. We should limit the options to just the descriptive names: "Sandbox", "Simulation", and "Production".

In such an approach, it no longer makes sense to call the field "Fatoora Server URL". Instead, we can introduce a new field called "Fatoora Server" and remove fatoora_server_url from JSON. We should leave fatoora_server_url in the database for now (remove it from JSON only) until a future patch once the change is widely deployed on production.